bool is_connect
string host
string port
string mountpoint
string username
string password
string ntrip_version
bool authenticate
---
bool success